common/lib/xmodule/xmodule/course_module.py

```diff
@@ -303,6 +303,12 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_("Enter the URL for the official course Facebook group. If you provide a URL, the edX mobile app includes a button that students can tap to access the group."),
```

Comment:
I thought before you had mentioned this being a Facebook Page. Is this intended for the course's group or page?